packages/generation/ir/ir-types/src/types.yml

```diff
@@ -7,13 +7,12 @@ types:
     docs: "A type, which is a name and a shape"
     extends: commons.WithDocs
     properties:
-      name: DeclaredTypeName
+      name: TypeName
       shape: Type
-  DeclaredTypeName:
+  TypeName:
```

Comment:
lets not do this -- it conflicts with java poet type name

Reply:
cool, thoughts on me changing errorname and servicename to match

Reply:
like `DeclaredErrorName` ?

Reply:
and `DeclaredServiceName` ?

Reply:
yeah